Sources/ActionSheet/ActionSheetCell.swift

```diff
@@ -90,6 +162,11 @@ class ActionSheetCell: UICollectionViewCell {
         let colors = PresentationTheme.current.colors
         name.textColor = isSelected ? colors.orangeUI : colors.cellTextColor
         tintColor = isSelected ? colors.orangeUI : colors.cellDetailTextColor
+        if accessoryType == .checkmark {
+            accessoryTypeImageView.isHidden = !isSelected
```

Comment:
isHidden has nothing to do with updateColors ^ that can be removed I believe.
We need to double-check here with Louis if the action sheet should always be darktheme styled in the movieviewcontroller, if so we need to make sure that this gets correctly called in the chevron case

Reply:
this will only trigger for the checkmark options so the colors of the chevron will always be the same tint of grey, in dark and light themes